src/initialize.js

```diff
@@ -46,6 +46,15 @@
  */
 
 import { createBrowserHistory, createMemoryHistory } from 'history';
+/*
+This 'env.config' package is a special 'magic' alias in our webpack configuration in frontend-build.
+It points at an `env.config.js` file in the root of an MFE's repository if it exists and falls back
+to an empty object `{}` if the file doesn't exist.  This acts like an 'optional' import, in a sense.
+Note that the env.config.js file in frontend-platform's root directory is NOT used by the actual
+initialization code, it's just there for the test suite and example application.
+*/
+import envConfig from 'env.config'; // eslint-disable-line import/no-unresolved
```

Comment:
[curious] Could we modify the default ESLint rules in `@edx/frontend-build` to disable the `import/no-unresolved` rule specifically for `env.config`? That way, we wouldn't need to disable the ESLint rule here. Given the file is exposed via `@edx/frontend-build`, it feels reasonable to have `@edx/frontend-build` make it pass ESLint, too.

Reply:
@davidjoy See https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md#ignore.